Makefile

```diff
@@ -7,6 +7,9 @@ ARTIFACT_DIR := $(if $(ARTIFACT_DIR),$(ARTIFACT_DIR),tests/test_results)
 TEST_TAGS := $(if $(TEST_TAGS),$(TEST_TAGS),"")
 SUITE_ID := $(if $(SUITE_ID),$(SUITE_ID),"nosuite")
 
+EVAL_MODEL := $(if $(EVAL_MODEL),$(EVAL_MODEL),"gpt")
```

Comment:
we already set provider+model env vars:
https://github.com/openshift/release/blob/7a74353dcd55f263e7755ee28f59b11c0b23812e/ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml#L68-L74

we should key off those, rather than introducing another env var that could end up out of sync w/ the ols config.